test/extended/cli/resources.go

```diff
@@ -0,0 +1,144 @@
+package cli
+
+import (
+	"github.com/openshift/origin/test/extended/util/image"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8simage "k8s.io/kubernetes/test/utils/image"
+)
+
+func newHelloPod() *corev1.Pod {
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "hello-openshift",
+			Labels: map[string]string{
+				"name": "hello-openshift",
+			},
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  "hello-openshift",
+					Image: k8simage.GetE2EImage(k8simage.Agnhost),
+					Args:  []string{"netexec"},
+					Ports: []corev1.ContainerPort{
+						{
+							ContainerPort: 8080,
+							Protocol:      corev1.ProtocolTCP,
+						},
+					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "tmp",
+							MountPath: "/tmp",
+						},
+					},
+					TerminationMessagePath: "/dev/termination-log",
+					ImagePullPolicy:        corev1.PullIfNotPresent,
+				},
+			},
+			Volumes: []corev1.Volume{
+				{
+					Name: "tmp",
+					VolumeSource: corev1.VolumeSource{
+						EmptyDir: &corev1.EmptyDirVolumeSource{},
+					},
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyAlways,
+			DNSPolicy:     corev1.DNSClusterFirst,
+		},
+	}
+}
+
+func newShellPod(shell string) *corev1.Pod {
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cli-test",
+		},
+		Spec: corev1.PodSpec{
+			ServiceAccountName: "builder",
```

Comment:
Don't use `builder` SA, I know it's been there before but that's not one we should use.